### ISSUE TITLE
Docs incorrectly say that vpc/vswitch for serverless cluster can be created automatically

### ISSUE THREAD

**Issue by jbg** [terraform-providers/terraform-provider-alicloud#2437]
> vpc_id - The vpc where new kubernetes cluster will be located. Specify one vpc's id, **if it is not specified, a new VPC will be built**.
> vswitch_id - The vswitch where new kubernetes cluster will be located. Specify one vswitch's id, **if it is not specified, a new VPC and VSwicth will be built**. It must be in the zone which availability_zone specified.

The above text appears on https://www.terraform.io/docs/providers/alicloud/r/cs_serverless_kubernetes.html (emphasis added by me)

However, if these are not specified, cluster creation fails with:

> Aliyun API Error: RequestId:  Status Code: 400 Code: InvalidParameters Message: The name,region_id,vpc_id,vswitch_id not allowed empty

**Comment by chushenmeshile** [comment]
@jbg They are required, if you don't, there will be mistakes in the implementation of the `terraform plan`

**Comment by jbg** [comment]
Yes, but the docs say they are not required. Docs should be fixed

**Comment by chushenmeshile** [comment]
@jbg Now it's fixed. And `vswitch_ids`  replace `vswitch_id`
[https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/cs_serverless_kubernetes#vpc_id](url)
